src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -419,6 +419,14 @@ public void MetricExtractorAfterSamplingVerbose(string appDomainName = "Incorrec
             this.WriteEvent(34, this.nameProvider.Name);
         }
 
+        [Event(35,
+            Message = "At least one telemetry channel cannot consume incoming telemetry fast enough. Some telemetry was dropped to prevent out of memory condition.",
```

Comment:
nitpick: "...prevent an out of memory condition."

Can we also include the number of items that were dropped and which channel it was that dropped them in the trace message?